crates/wasmtime/src/runtime/func.rs

```diff
@@ -2109,6 +2109,26 @@ impl<T> Caller<'_, T> {
             .get_export(&mut self.store, name)
     }
 
+    /// Looks up an exported [`Extern`] value by a [`ModuleExport`] value.
+    ///
+    /// This is similar to [`Self::get_export`] but uses a [`ModuleExport`] value to avoid
+    /// string lookups where possible. [`ModuleExport`]s can be obtained by calling
+    /// [`Module::get_export_index`] on the [`Module`] that an instance was instantiated with.
+    ///
+    /// This method will search the module for an export with a matching entity index and return
+    /// the value, if found.
+    ///
+    /// Returns `None` if there was no export with a matching entity index.
+    /// # Panics
+    ///
+    /// Panics if `store` does not own this instance.
```

Comment:
Docs are effectively a copy/paste of `Instance::get_module_export`. Happy to take recommendations or if you want to make a suggestion, I will definitely accept it. I think `see [Instance::get_module_export] for details` would probably also be fine so long as it's clear that the `caller` has access to the instance (wasn't clear to me without looking at the code).